SwiftUIDZ/SwiftUIDZ/ContentView.swift

```diff
@@ -7,15 +7,207 @@
+    @State private var cellularCommunicationSelection = Constants.zeroNumber
+    @State private var modemSelection = Constants.zeroNumber
+    
+    private var wifiStatuses = Constants.connectionStatuses
+    private var switchedStatuses = Constants.switchedStatuses
+    private var cellularCommunicationStatuses = Constants.cellularCommunicationStatuses
+
     var body: some View {
-        VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+        NavigationView {
+            Form {
+                Section {
+                    makeProfileContainer()
+                    makeOffersAppleIdContainer()
+                }
+                Section {
+                    makeUpdateContainer()
+                }
+                Section {
+                    makeAirplaneModeContainer()
+                    makeWiFiContainer()
+                    makeBluetoothContainer()
+                    makeCellularCommunicationContainer()
+                    makeModemContainer()
+                    makeVPNContainer()
+                }
+            }
+            .navigationTitle(Constants.navigationTitleText)
+        }
+    }
+    
+    private func makeProfileContainer() -> some View {
+        HStack {
+                Image(Constants.bearImageName)
+                    .resizable()
+                    .frame(width: Constants.profileImageSide, height: Constants.profileImageSide)
+                    .cornerRadius(CGFloat(Constants.profileImageCornerRadius))
+            VStack {
+                    Text(Constants.userName)
+                    .offset(x: Constants.userNameShadowOffsetX, y: Constants.userNameShadowOffsetY)
+                        .font(.system(size: Constants.userFontSizeNumber))
+                Spacer(minLength: Constants.minLengthSpacerUserContainer)
+                    Text(Constants.profileInfoText)
+                    .font(.system(size: Constants.userDescriptionFontSizeNumber)).offset(x: Constants.userDescriptionShadowOffsetX)
+                    Spacer()
+            }
+            Spacer()
+            Image(systemName: Constants.chevronForwardSystemImageName)
+                .foregroundColor(.gray)
+        }
+    }
+    
+    fileprivate func makeOffersAppleIdContainer() -> some View {
+        HStack {
+            Text(Constants.offersAppleIdText)
+            Spacer()
+            ZStack {
+                Circle()
+                    .foregroundColor(.red)
+                    .frame(width: Constants.offersSideNumber, height: Constants.offersSideNumber)
+                Text(Constants.offersAppleIdCount)
+                    .foregroundColor(.white)
+            }
+            Image(systemName: Constants.chevronForwardSystemImageName)
+                .foregroundColor(.gray)
+        }
+    }
+    
+    fileprivate func makeUpdateContainer() -> some View {
```

Comment:
```suggestion
    private func makeUpdateContainer() -> some View {
```
Везде где нет параметров переделай из метода в вычисляемое и добавь в конце View